plugins/reporters/freemarker/src/main/kotlin/FreemarkerTemplateProcessor.kt

```diff
@@ -291,26 +284,23 @@ class FreemarkerTemplateProcessor(
         @JvmOverloads
         @Suppress("UNUSED") // This function is used in the templates.
         fun hasUnresolvedRuleViolations(threshold: Severity = input.ortConfig.severeRuleViolationThreshold) =
-            input.ortResult.evaluator?.violations?.any { violation ->
-                val isResolved = input.resolutionProvider.isResolved(violation)
-                val isExcluded = violation.pkg?.let { input.ortResult.isExcluded(it) } ?: false
-
-                violation.severity >= threshold && !isResolved && !isExcluded
-            } ?: false
+            input.ortResult.getRuleViolations(omitResolved = true, minSeverity = threshold).any { violation ->
+                violation.pkg?.let { input.ortResult.isExcluded(it) } ?: false
+            }
 
         /**
          * Return a list of [RuleViolation]s for which no [RuleViolationResolution] is provided.
          */
         @Suppress("UNUSED") // This function is used in the templates.
         fun filterForUnresolvedRuleViolations(ruleViolation: List<RuleViolation>): List<RuleViolation> =
-            ruleViolation.filterNot { input.resolutionProvider.isResolved(it) }
+            ruleViolation.filterNot { input.ortResult.isResolved(it) }
```

Comment:
I wonder whether we should instead simply drop these functions and tell reporter authors to call `getRuleViolations(omitResolved = true)`. Similar below.

Reply:
I prefer this diff as-is in this commit. 
I'll add a commit on top of the PR to inline the functions.

Reply:
This would be a breaking change, as we basically don't know who is making use of these.
I strongly prefer to, if at all, do this out of scope of this PR in a separate dedicated PR later on.